Client/game_sa/CStreamingSA.cpp

```diff
@@ -247,3 +251,47 @@ void CStreamingSA::RemoveArchive(unsigned char ucArhiveID)
     CloseHandle(m_aStreamingHandlers[uiStreamHandlerID]);
     m_aStreamingHandlers[uiStreamHandlerID] = NULL;
 }
+
+void CStreamingSA::SetStreamingBufferSize(uint32 uiBlockSize)
+{
+    if (uiBlockSize <= ms_streamingBufferSize * 2)
+        return;
+
+    // Close old streaming handle
+    TerminateThread(*phStreamingThread, 0);
+
+    // Create new buffer
+    if (uiBlockSize & 1)
+        uiBlockSize = uiBlockSize++ + 1;
+
+    typedef int(__cdecl * Function_CMemoryMgr_MallocAlign)(uint32 uiCount, uint32 uiAlign);
+    int iPointer = ((Function_CMemoryMgr_MallocAlign)(0x72F4C0))(uiBlockSize << 11, 2048);
```

Comment:
use `void*`

Reply:
yeah, but we do some math operations with `iPointer`

Reply:
Do it like in the original code.
https://github.com/codenulls/gta-reversed/blob/d5e83be9f188daf57902d0199f6c5019ba6f3e8d/source/game_sa/CStreaming.cpp#L2745-L2747

Reply:
This code don't uses `void*` also

Reply:
void* used